docs/tables/kubernetes_pod_disruption_budget.md

```diff
@@ -0,0 +1,37 @@
+# Table: kubernetes_pod_disruption_budget
+
+A Pod Disruption Budgeti (PDB) limits the number of Pods of a replicated application that are down simultaneously from voluntary disruptions.
+
+## Examples
+
+### Basic Info
+
+```sql
+select
+  name,
+  namespace,
+  min_available,
+  max_unavailable,
+  selector
+from
+  kubernetes_pod_disruption_budget
+order by
+  namespace, name;
+```
+
```

Comment:
Please add some more example queries around minAvailable/maxUnavailable, for reference you can check [this](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) document.